defaults/config.js

```diff
@@ -35,12 +35,57 @@ module.exports = {
 	bind: undefined,
 
 	//
-	// Set the default theme.
+	// Themes
 	//
-	// @type     string
-	// @default  "themes/example.css"
+	// @type     array
 	//
-	theme: "themes/example.css",
+	themes: [
+		//
+		// A theme
+		//
+		// @type     object
+		//
+		{
+			//
+			// Name
+			//
+			// @type     string
+			// @default  "Default"
+			//
+      		"name": "Default",
```

Comment:
@astorije I was just following what the rest of this file uses. It is all tabs, though that might have changed in the last 5 months.